src/Microsoft.AspNetCore.WebUtilities/MultipartSectionConverterExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.WebUtilities
+{
+    /// <summary>
+    /// Various extensions for converting multipart sections
+    /// </summary>
+    public static class MultipartSectionConverterExtensions
+    {
+        /// <summary>
+        /// Converts the section to a file section
+        /// </summary>
+        /// <param name="section">The section to convert</param>
+        /// <returns>A file section</returns>
+        public static FileMultipartSection AsFileSection(this MultipartSection section)
```

Comment:
Is there any official guideline for `As<>`? What would you expect to get back, a null?